### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by rbanos-mv**
# React tutorial
[React Tutorial From Scratch: A Step-by-Step Guide (2021).](https://ibaslogic.com/react-tutorial-for-beginners/)
This tutorial, divided into several parts, is very comprehensive, and covers all main concepts about React, from the very basics to some intermediate topics. Feel free to skip the parts you think you don't need, since you already learned about these topics. This tutorial is not a substitute, just a reinforcement of what you have learned.

**The tutorial is up and running.** [Live Demo Link](https://rbanos-mv.github.io/react-todo-app/)

### General requirements
- Make sure that there are no linter errors.
- Make sure that you used correct Gitflow.
- Make sure that you documented your work in a professional way.
### JavaScript requirements
- Follow our list of best practices for JavaScript.
### Project requirements
- Part 1 - React tutorial: The beginner's guide to learning React in 2020.
  - Setup a ReactJs project
  - Create components: TodoContainer, TodoList, TodoItem, InputTodo, Header
  - Add State
- Part 2 - Working with React form and handling event.
  - Add checkboxes
  - Raising and handling events
  - Updating the state using the setState() method
  - Using the setState updater
  - Deleting items from the todos
  - Adding a text input field and a submit button
  - Updating the Todos list
  - Generating random ids for the todos list items
  - Validating the Form Submission
- Part 3 – How to implement CSS in Reactjs app.
  - Styling with CSS stylesheet
  - Inline styling
  - Styling with CSS Modules
  - Styling completed todos items
  - Using Destructuring
- Part 4 – How to edit to dos items.
  - Edit todo item on double click
  - Detecting when the Users press the Enter key to submit edited Items

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,66 @@
-# react-todo-app
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> Hello! I am a software developer! I can help you build a product, feature or website. Tak a look of my works.
```
Comment (wandergithub) [inline README.md]: [Optional] Kindly suggest modifying the project description. Or just removing this part. Looks like is part of your old project template.

**Review by wandergithub — APPROVED** [review]
# Hi @rbanos-mv 👋🏽 

Your project is complete!  There is nothing else to say other than… it's time to merge it.
Congratulations! 🎉 

## To highlight 🎯
- No linter errors.✔️
- Good code design and implementation. ✔️
-  Everything works and looks good on the browser.✔️ 
- Good documentation. ✔️ 


## Optional suggestions

Please also take a look at the comments. ⬆️ 

[Optional] Kindly suggest removing boilerplate files, as they are no longer required and that would make your workspace look cleaner.
image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question, so I can receive the notification.

File: README.md [hunk 2]
```diff
@@ -1 +1,66 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```
Comment (wandergithub) [inline README.md]: Please Remove template code.

Comment (rbanos-mv) [reply README.md]: I added an acknowledgment the author of the tutorial.

File: README.md (hunk 1 above)
Comment (rbanos-mv) [reply README.md]: @wandergithub Done.

File: README.md (hunk 2 above)
Comment (rbanos-mv) [reply README.md]: @wandergithub Boilerplate files were removed.

**Pull request merged**